<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalStudy</class>
 <widget class="QWidget" name="SignalStudy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Zählrichtung</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="backwardButton">
          <property name="text">
           <string>rückwärts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="forwardButton">
          <property name="text">
           <string>vorwärts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="extensionBox">
          <property name="maximum">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Abschnitt</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="freeButton">
          <property name="text">
           <string>frei</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="shuntButton">
          <property name="text">
           <string>Rangieren</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tourButton">
          <property name="text">
           <string>Fahrt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="occupiedButton">
          <property name="text">
           <string>belegt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="passedButton">
          <property name="text">
           <string>durchgefahren</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="title">
        <string>Rangiersignal</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="availableButton_1">
          <property name="text">
           <string>verfügbar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="stopButton_1">
          <property name="text">
           <string>Halt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="goButton_1">
          <property name="text">
           <string>Fahrt</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Vorsignal</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="availableButton_2">
          <property name="text">
           <string>verfügbar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="stopButton_2">
          <property name="text">
           <string>Halt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="goButton_2">
          <property name="text">
           <string>Fahrt</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Hauptsignal</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="availableButton_3">
          <property name="text">
           <string>verfügbar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="stopButton_3">
          <property name="text">
           <string>Halt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="goButton_3">
          <property name="text">
           <string>Fahrt</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="symbolWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="mrw::ui::SignalWidget" name="smallSwitchWidget" native="true">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="mrw::ui::SignalWidget" name="bigSwitchWidget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mrw::ui::SignalWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/signalwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>backwardButton</tabstop>
  <tabstop>forwardButton</tabstop>
  <tabstop>extensionBox</tabstop>
  <tabstop>freeButton</tabstop>
  <tabstop>shuntButton</tabstop>
  <tabstop>tourButton</tabstop>
  <tabstop>occupiedButton</tabstop>
  <tabstop>passedButton</tabstop>
  <tabstop>availableButton_1</tabstop>
  <tabstop>stopButton_1</tabstop>
  <tabstop>goButton_1</tabstop>
  <tabstop>availableButton_2</tabstop>
  <tabstop>stopButton_2</tabstop>
  <tabstop>goButton_2</tabstop>
  <tabstop>availableButton_3</tabstop>
  <tabstop>stopButton_3</tabstop>
  <tabstop>goButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
